src/OpenTelemetry.Api/Trace/Tracer.cs

```diff
@@ -35,6 +35,14 @@ internal Tracer(ActivitySource activitySource)
         this.ActivitySource = activitySource;
     }
 
+    [Flags]
+    private enum StartSpanBehaviors
+    {
+        Active = 0b1,
+        Inactive = 0b10,
+        Root = 0b100,
```

Comment:
very nitpicking on my part, but, I think it will be easier to read if it was something like:
```suggestion
        ActivateNewSpan = 0b1,
        DeactivateNewSpan = 0b10,
        NewSpanAsRoot = 0b100,
```

Reply:
Updated